src/oneD/StFlow.cpp

```diff
@@ -1027,12 +1027,11 @@ void StFlow::evalRightBoundary(double* x, double* rsd, int* diag, double rdt)
     diag[index(c_offset_Y + rightExcessSpecies(), j)] = 0;
     if (m_usesLambda) {
         rsd[index(c_offset_U, j)] = rho_u(x, j);
-        rsd[index(c_offset_L, j)] = lambda(x, j);
     } else {
         rsd[index(c_offset_U, j)] = rho_u(x, j) - rho_u(x, j-1);
-        rsd[index(c_offset_L, j)] = lambda(x, j) - lambda(x, j-1);
```

Comment:
```suggestion
        rsd[index(c_offset_L, j)] = lambda(x, j);
```
zero (lambda is not used)